src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -26,10 +26,12 @@ ALTER SEQUENCE pg_catalog.pg_dist_placement_placementid_seq RESTART 100000;
 SELECT nextval('pg_catalog.pg_dist_groupid_seq') AS last_group_id \gset
 SELECT nextval('pg_catalog.pg_dist_node_nodeid_seq') AS last_node_id \gset
 -- Create the necessary test utility function
+SET citus.enable_metadata_sync TO OFF;
```

Comment:
why do we set this off?

Reply:
To not to have those deprecated function on other test outputs.